sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/Generated/ConfidentialLedgerIdentityServiceClient.cs

```diff
@@ -31,6 +31,30 @@ protected ConfidentialLedgerIdentityServiceClient()
         {
         }
 
+        /// <summary> Initializes a new instance of ConfidentialLedgerIdentityServiceClient. </summary>
+        /// <param name="identityServiceUri"> The Identity Service URL, for example https://identity.accledger.azure.com. </param>
+        /// <param name="credential"> A credential used to authenticate to an Azure Service. </param>
+        /// <param name="options"> The options for configuring the client. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="identityServiceUri"/> or <paramref name="credential"/> is null. </exception>
+        public ConfidentialLedgerIdentityServiceClient(Uri identityServiceUri, TokenCredential credential, ConfidentialLedgerClientsOptions options = null)
```

Comment:
The rename of the options type caused a duplicate of this ctor
https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/ConfidentialLedgerIdentityServiceClient.cs#L20

Reply:
Fixed